### PR TITLE
release notes for 2.0.5

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.7 only
- [Preview](https://deploy-preview-44432--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-2-x)
 ptal

### CODE REVIEW

**Comment by alinaryan** [comment]
LGTM

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 1]
```diff
@@ -21,15 +21,23 @@ ifndef::openshift-origin[]
 
 Windows Container Support for Red Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
 
-You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
```
Comment (mjpytlak) [inline windows_containers/windows-containers-release-notes-2-x.adoc:24]:
```suggestion
You must have a separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
```

**Comment by mjpytlak** [comment]
@Amrita42 just a few nits. Also, I wanted to confirm that you want this merged. I noticed that 2.0.5 is not shipping until 5/22. Thank you.

**Comment by alinaryan** [comment]
@mjpytlak @Amrita42 
2.0.5 is shipping on 5/02, not 5/22.

**Comment by mjpytlak** [comment]
> @mjpytlak @Amrita42 2.0.5 is shipping on 5/02, not 5/22.

Thanks for the clarification. I misread the entry. That is still nearly 3 weeks away. Do you want to merge after addressing the feedback or hold until the release date @Amrita42?

**Comment by mjpytlak** [comment]
Peer review LGTM.

**Pull request merged**